packages/perseus/src/server-item-renderer.tsx

```diff
@@ -60,10 +60,24 @@ type DefaultProps = Required<
 >;
 
 type State = {
+    /**
+     * questionCompleted is used by Radio and LabelImage to change what
+     * it renders after a learner has checked the correctness of an answer.
+     * For instance LabelImage disables inputs for correct answers and marks
+     * incorrect answers so learners can update what they submitted.
+     */
     questionCompleted: boolean;
+    /**
+     * As far as I can tell, this is used to highlight empty widgets
+     * after a learner has clicked the "check" button. I don't think this could
+     * still be used though, because the "check" button is disabled while there
+     * are empty widgets.
+     */
     questionHighlightedWidgets: ReadonlyArray<any>;
```

Comment:
That's correct.  And to be really clear, it's an array of widget IDs (strings). Could you try changing this to `ReadonlyArray<string>`